web/viewer.css

```diff
@@ -135,32 +135,6 @@ select {
   cursor: default;
 }
 
-/* outer/inner center provides horizontal center */
-.outerCenter {
-  pointer-events: none;
```

Comment:
*Note for myself:* Check that removing `pointer-events: none;` doesn't accidentally regress PR #3671.